### ISSUE TITLE
Board Review: Azure Notification Hubs Web Push

### ISSUE THREAD

**Issue by mpodwysocki** [Azure/azure-sdk#5664]
## Contacts and Timeline

* Responsible service team: Azure Notification Hubs
* Main contacts: matthewp@microsoft.com
* Expected code complete date: 3/2023
* Expected release date: 3/2023

## About the Service 

* Link to documentation introducing/describing the service: https://learn.microsoft.com/en-us/azure/notification-hubs/
* Link to the service REST APIs: https://aka.ms/nh-rest-api 
* Link to GitHub issue for previous review sessions, if applicable:  #4985

## About the client library

* Name of the client library: Azure Notification Hubs Web Push Client SDK
* Languages for this review: JavaScript/TypeScript

## Artifacts required (per language)

Please read through “API Review” section [here](https://azure.github.io/azure-sdk/policies_reviewprocess.html) to understand how these artifacts are generated. **It is critical that these artifacts are present and are in the right format. If not, the language architects cannot review them with the SDK Team’s API review tool.**
  
### TypeScript

* APIView Link: https://apiview.dev/Assemblies/Review/fee7730ab5de481b9f652643c3dba4e4
* Link to Champion Scenarios/Quickstart samples: https://github.com/Azure/azure-sdk-for-js/tree/feat/nh-web-push/sdk/notificationhubs/notification-hubs-web-push/samples-dev

**Comment by ronniegeraghty** [comment]
@mpodwysocki, is this issue for the same topic as this one: #5669?

**Comment by mpodwysocki** [comment]
@ronniegeraghty no, they are two separate topics, but could be combined into the same meeting if possible
